packages/core/src/DelegateMixin.js

```diff
@@ -18,6 +18,7 @@ import { dedupeMixin } from '@open-wc/dedupe-mixin';
  */
 const DelegateMixinImplementation = superclass =>
   // eslint-disable-next-line
+  // @ts-ignore https://github.com/microsoft/TypeScript/issues/36821#issuecomment-588375051
   class extends superclass {
```

Comment:
Will there be an alternative way, so we can become "skipLibCheck free" again?

Reply:
TS and Mixins will probably not be friends any time soon 😅

for `DelegateMixin` in particular, it could be a lit controller... and this pattern is supported by TS
IMHO we should convert all mixins we can to lit controllers

I expect quite some Mixins will remain unless we find a nice way to add properties via controllers... so then it's either skipLibCheck or do specific inheritance...

Reply:
Yeah I think the only way to be skipLibCheck free again is for us to migrate to something typescript can understand, like Lit controllers. I'm somewhat skeptical that this will magically solve our typed inheritance issue that surfaced with the latest typescript, but who knows.

My interpretation of the comments by Ryan Cavanaugh are that @ts-ignore is not a sin and not uncommon, which means that skipLibCheck as an extension of that isn't either. So when users ask about it we can just explain that or have it in our README -> Rationale somewhere?